### DISCUSSION TITLE
Giada Multi-channels?

### DISCUSSION THREAD

**Discussion by amelieppe** [monocasual/giada#689] [Ideas]
How complicated would it be to get giada to play on multiple channels rather than stereo?
It would be really great if we could affect either :
- the desired outputs for each sample 
- the desired outputs for each column.
Thanks in any case for your answers, and your work!
Amélie

**Comment by gvnnz** [comment]
Hi @amelieppe, thanks for your input. Yes, multi-channel output is an important feature we want to add in Giada, and there's entire milestone for that: https://github.com/monocasual/giada/milestone/30!
